### ISSUE TITLE
epub failed: pandoc does not reconize the `--atx-headers` options

### ISSUE THREAD

**Issue by g-berthiaume** [phiresky/ripgrep-all#189]
**Describe the bug**
I'm trying to use `rga` to look at an epub.
 `rga` seems to be trying to use `pandoc` with an invalid flag.
This seems to be the case for every epub I'm trying on my machine.

**To Reproduce**

```powershell
$ rga "my-keyword" .

.\test.epub: preprocessor command failed: '"C:\\mytest\\rga-preproc" ".\\test.epub"':
-------------------------------------------------------------------------------
adapter: pandoc
Unknown option --atx-headers.
Try pandoc.exe --help for more information.
Error: The pipe has been ended. (os error 109)
-------------------------------------------------------------------------------
```

**Operating System and Version**
![image](https://github.com/phiresky/ripgrep-all/assets/19377511/357a63cb-7b98-4c08-a80e-65ac00b98e0f)

Thanks for rga, it's a great tool!

**Comment by phiresky** [comment]
This is fixed in 1.0.0-beta

**Comment by g-berthiaume** [comment]
Hi @phiresky, 
Thanks for your answer. 
I'll wait for the beta release.

Have a good day!

**Comment by lafrenierejm** [comment]
> This is fixed in 1.0.0-beta

@phiresky Heads up that I don't see any tag or branch for a beta release. The latest I'm seeing is [`v1.0.0-alpha.5`](https://github.com/phiresky/ripgrep-all/releases/tag/v1.0.0-alpha.5).

**Comment by phiresky** [comment]
True, I misspoke. This is fixed in the alpha release.
